docs/painless/painless-operators-numeric.asciidoc

```diff
@@ -0,0 +1,663 @@
+[[painless-operators-numeric]]
+=== Operators: Numeric
+
+[[post-increment-operator]]
+==== Post Increment
+
+A variable/field representing a numerical value can be possibly evaluated as part of an expression, and then increased by 1 for its respective type.  The format starts with a variable name followed by a plus and ends with a plus.
+
```

Comment:
Should follow the same format used for the other operators--something like, "use the post increment operator (++) to increase the value of a variable by one after it has been evaluated. The format is the variable name followed by `++`.

Reply:
Changed.